### PR TITLE
fix: remove bottom margin

### PR DESCRIPTION
**Pull request by PaulFarault**
#### Which issue(s) this PR fixes

Fixes None

#### Agreements

- [x] I hereby declare this contribution to be licensed under the [Apache License Version 2.0, January 2004](http://www.apache.org/licenses/LICENSE-2.0).
- [x] I hereby agree to grant [TOSIT](https://www.tosit.io/) a copyright license to use my contributions.

### CODE REVIEW

**Review by jpsavy — APPROVED** [review]
The bottom margin is removed.

File: src/components/Layout/Layout.tsx [hunk 1]
```diff
@@ -23,7 +23,7 @@ export function DashboardLayout({ children }) {
           </div>
         </aside>
         {/* Main content */}
-        <div className="h-screen overflow-y-auto flex-col mb-3">
+        <div className="h-screen overflow-y-auto flex-col">
```
Comment (pvalentin01) [inline src/components/Layout/Layout.tsx:26]: flex-col may be used with flex, otherwise it has no effect

**Comment by pvalentin01** [comment]
One less vertical scrollbar

File: src/components/Layout/Layout.tsx (hunk 1 above)
Comment (PaulFarault) [reply src/components/Layout/Layout.tsx:26]:
```suggestion
        <div className="h-screen overflow-y-auto">
```

You're right, that wasn't the aim of this PR but we can remove it too.

**Review by pvalentin01 — APPROVED** [review]

**Pull request merged**